dashboard/src/shared/Auth.ts

```diff
@@ -3,7 +3,7 @@
 
 import { grpc } from "@improbable-eng/grpc-web";
 import { AxiosResponse } from "axios";
-import * as jwt from "jsonwebtoken";
+import jwt from "jsonwebtoken";
```

Comment:
See https://github.com/auth0/node-jsonwebtoken/issues/875, I don't think this change should affect us, but let's see what CI thinks.

At some point, we should move from `jwt.decode` to `jwt.verify` 😅